generators/server/templates/sql/reactive/src/main/java/package/repository/EntityManager.java.ejs

```diff
@@ -157,7 +157,7 @@ public class EntityManager {
      * @param entityType the entity type which holds the table name.
      * @return the number of deleted entity
      */
-    public Mono<Integer> deleteAll(Class<?> entityType) {
+    public Mono<Long> deleteAll(Class<?> entityType) {
```

Comment:
Why is this change necessary? It seems unrelated to Elasticsearch. I'm fine with it, but not sure if it'll mess up database logic.

Reply:
In order to fix the reactive elasticsearch client I generated a reactive project.
Together with the replacement of the right package for `ReactiveElasticsearchTemplate`, I changed also the methods in this class because in several case (like this one) the latest version (`3.0.0-M6`) of `spring-data-r2dbc`  return `Mono<Integer>` instead of `Mono<Long>`.

I think this changed is necessary for the migration to Spring Boot 3, but as you wrote it is unrelated to the elastichseach.
Therefore if you want I can undo just the changes in the `EntitManager` class.

Reply:
I think it's OK to make this change.